vnext/PropertySheets/FastBuild.targets

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
>Project [](http://example.com/codeflow?start=1&length=7)

How is this target file used by projects?

Reply:
The target will add the "RNW_FASTBUILD" definition based on the amount of memory per thread available on the system.

Reply:
Should we have project files or other common targets files reference this targets file?

Reply:
@vmoroz: I believe that the RNW_FASTBUILD is only used by `vnext/Microsoft.ReactNative/Pch/pch.h` so it makes sense to keep this here.. If we add it to other pch files to speed up those builds then I agreen we should move it to Directory.Build.Props